saylua/modules/messages/templates/messages/all.html

```diff
@@ -44,10 +44,10 @@
         Status
       </th>
     </tr>
-    {% for message in viewed_messages %}
-    <tr class="forum-board-row{% if not message.is_read %} highlight{% endif %}">
+    {% for message in messages %}
+    <tr class="forum-board-row{% if not message.unread %} highlight{% endif %}">
       <td>
-        <input type="checkbox" name="user_conversation_id" value="{{ message.key.urlsafe() }}">
+        <input type="checkbox" name="user_conversation_id" value="{{ message.conversation_id }}">
```

Comment:
This is for bulk actions on messages. ie: you can select 5 messages in your inbox and delete them.